lib/auth/touchid/api.go

```diff
@@ -373,3 +380,38 @@ func Login(origin, user string, assertion *wanlib.CredentialAssertion) (*wanlib.
 		},
 	}, cred.User, nil
 }
+
+// ListCredentials lists all registered Secure Enclave credentials.
+// Requires user interaction.
+func ListCredentials() ([]CredentialInfo, error) {
+	if !native.IsAvailable() {
+		return nil, ErrNotAvailable
+	}
```

Comment:
It's a bit overly cautious. I have some plans to stick additional checks on IsAvailable, in which case this starts to make more sense, but since I delayed that we can remove for now.